tests/integration/cattletest/core/test_network_policy.py

```diff
@@ -0,0 +1,54 @@
+from common_fixtures import *  # NOQA
+
+
+def test_network_policy_default_action(super_client):
+    nd = super_client.wait_success(super_client.create_network_driver())
+    network = super_client.create_network(isPublic=True,
+                                          networkDriverId=nd.id)
+    network = super_client.wait_success(network)
+    assert network.state == 'active'
+    assert network.defaultPolicyAction == 'allow'
+
+    return network
+
+
+def test_default_no_network_policy(super_client):
+    nd = super_client.wait_success(super_client.create_network_driver())
+    network = super_client.create_network(isPublic=True,
+                                          networkDriverId=nd.id)
+    network = super_client.wait_success(network)
+    assert network.state == 'active'
+    assert network.policy == None
+
+    return network
+
+
+def test_default_network_policy(super_client):
+    nd = super_client.wait_success(super_client.create_network_driver())
+    network = super_client.create_network(isPublic=True,
+                                          networkDriverId=nd.id,
+                                          defaultPolicyAction='deny',
+                                          policy=['{"within": "stack", "action": "allow"}'])
+    network = super_client.wait_success(network)
+    assert network.state == 'active'
+    assert network.defaultPolicyAction == 'deny'
+    assert len(network.policy) == 1
```

Comment:
can we also add a validation for network.policy fields? Just to make sure its every field/value was set